openquakeplatform/openquakeplatform/templates/irv_viewer.html

```diff
@@ -45,9 +45,7 @@
     <li id="1" ><a href="#project-def">Project Definition</a></li>
     <li id="2"><a href="#iri-chart">IRI Chart</a></li>
     <li id="3"><a href="#cat-chart">SVI Theme Chart</a></li>
-    <!-- remove this feature temporarily
     <li id="4"><a href="#primary-tab">Primary Chart</a></li>
```

Comment:
We were thinking about renaming Primary Chart into something like Primary Indicators Chart